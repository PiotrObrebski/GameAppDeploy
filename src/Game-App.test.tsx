import { shallow, ShallowWrapper } from 'enzyme';
import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';

import GameApp from './Game-App';
import store from './redux/store';

describe("GameApp", () => {
  let component: ShallowWrapper<{ any: any }>;

  beforeEach(() => {
    component = shallow(<GameApp />);
  });

  test("Should mount", () => {
    expect(component.length).toBe(1);
  });

  test("should match snapshot", () => {
    const GameAppRender = renderer
      .create(
        <Provider store={store}>
          <GameApp />
        </Provider>
      )
      .toJSON();
    expect(GameAppRender).toMatchSnapshot();
  });
});
