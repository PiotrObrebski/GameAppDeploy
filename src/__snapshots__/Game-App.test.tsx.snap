// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameApp should match snapshot 1`] = `
<div
  className="game-app"
>
  <div
    className="wrapper"
  >
    <div
      className="landing-page"
    />
  </div>
</div>
`;
